config.yaml

```
@@ -2204,23 +2204,28 @@ Images:
   - v0.14.0
   - v0.15.0
   - v0.15.1
-- SourceImage: quay.io/coreos/kube-state-metrics
+- DoNotMirror: true
```

Maybe add a note as to why?
```suggestion
- DoNotMirror: true  # images have been removed from source
```